SECURITY-INSIGHTS.yml

```diff
@@ -0,0 +1,73 @@
+header:
+  expiration-date: '2025-07-04T00:00:00.000Z'
+  project-url: 'https://github.com/karmada-io/karmada'
+  schema-version: '1.0.0'
+  last-updated: '2024-07-04'
+  last-reviewed: '2024-07-04'
+  changelog: 'https://github.com/karmada-io/karmada/tree/master/docs/CHANGELOG'
+  license: 'https://github.com/karmada-io/karmada/blob/master/LICENSE'
+project-lifecycle:
+  status: 'active'
+  bug-fixes-only: False
+  core-maintainers:
+    - 'https://github.com/karmada-io/karmada/blob/master/MAINTAINERS.md'
+  roadmap: 'https://github.com/karmada-io/karmada/blob/master/ROADMAP.md'
+  release-process: 'https://karmada.io/docs/releases'
+contribution-policy:
+  accepts-pull-requests: True
+  accepts-automated-pull-requests: True
+  automated-tools-list:
+    - automated-tool: 'dependabot'
+      action: 'allowed'
+      path:
+        - /
+        - /cluster/images/
+  contributing-policy: 'https://github.com/karmada-io/community/blob/main/CONTRIBUTING.md'
+  code-of-conduct: 'https://github.com/karmada-io/community/blob/main/CODE_OF_CONDUCT.md'
+documentation:
+  - 'https://karmada.io/'
+distribution-points:
+  - 'https://hub.docker.com/u/karmada'
+  - 'https://github.com/karmada-io/karmada/releases'
+security-testing:
+  - tool-type: 'sca'
+    tool-name: 'Dependabot'
+    tool-version: '2'
+    integration:
+      ad-hoc: False
+      ci: False
+      before-release: False
+    tool-rulesets:
+      - 'https://github.com/karmada-io/karmada/blob/master/.github/dependabot.yml'
+    tool-url: 'https://github.com/dependabot'
+  - tool-type: 'sca'
+    tool-name: 'trivy'
+    tool-version: 'v0.52.2'
+    integration:
+      ad-hoc: False
+      ci: True
+      before-release: False
+    tool-rulesets:
+      - 'https://github.com/karmada-io/karmada/blob/master/.github/workflows/ci-image-scanning.yaml'
+    tool-url: 'https://github.com/aquasecurity/trivy-action'
+    comment: 'trivy-action is the github action for trivy. The current version of trivy-acton is v0.23.0 and the version of trivy used is v0.52.2.'
+security-contacts:
+  - type: 'email'
+    value: 'wangzefeng@huawei.com'
+  - type: 'email'
+    value: 'renhongcai@huawei.com'
+vulnerability-reporting:
+  accepts-vulnerability-reports: True
+  email-contact: 'renhongcai@huawei.com'
+  security-policy: 'https://github.com/karmada-io/community/blob/main/security-team/SECURITY.md#security-policy'
+dependencies:
+  dependencies-lists:
+    - 'https://github.com/karmada-io/karmada/blob/master/go.mod'
+    - 'https://github.com/karmada-io/karmada/blob/master/.go-version'
+    - 'https://github.com/karmada-io/karmada/blob/master/cluster/images/Dockerfile'
+  third-party-packages: True
+  sbom:
+    - sbom-file: 'https://github.com/karmada-io/karmada/releases' # Starting from v1.10.2, every release's assets include SBOMs.
+      sbom-format: 'SPDX'
+  env-dependencies-policy:
+    policy-url: 'https://github.com/karmada-io/community/blob/main/security-team/SECURITY.md#dependencies-policy'
```

Comment:
refer to https://github.com/karmada-io/community/pull/77

Reply:
I'll look at this dependencies-policy.